src/openzaak/components/zaken/api/utils.py

```diff
@@ -1,7 +1,9 @@
 # SPDX-License-Identifier: EUPL-1.2
 # Copyright (C) 2019 - 2020 Dimpact
 from vng_api_common.utils import get_uuid_from_path
-from zgw_consumers.client import UnknownService
+
+# TODO: remove legacy import of ZGWClient
+from zgw_consumers.legacy.client import UnknownService
```

Comment:
I checked this, it seems that we do raise `UnknownService` in a couple of places, but we don't catch it anywhere, so we do not rely on it actually being `zgw_consumers.legacy.client.UnknownService`. We could maybe replace it with our own exception (like OF does https://github.com/open-formulieren/open-forms/blob/262d84e52955a4dd86c4d41a4ae85a484a79b8cf/src/openforms/registrations/contrib/zgw_apis/client.py#L19) to get rid of the legacy import